.git-blame-ignore-revs

```diff
@@ -0,0 +1 @@
+58d9cf0b3a916af3e48fbb63b85b699c998c7f7a
```

Comment:
FYI a rebase and merge will change the relevant commit number

Reply:
(at least it will assuming it actually does a rebase... if it's already up to date maybe it won't?)

Reply:
oh damn, so I should create a merge commit? Thanks!